README.md

```diff
@@ -17,6 +17,180 @@ Copyright (C) 2020 National Science Foundation - National Center for Atmospheric
+```
+from acom_music_box import MusicBox, Conditions
+import musica.mechanism_configuration as mc                                      
+```
+
+2. Define the chemical system of interest
+
+MusicBox uses [Musica](https://ncar.github.io/musica/index.html) to create specific chemical species and phases of interest for chemical mechanisms.
+```
+A = mc.Species(name="A")
+B = mc.Species(name="B")
+C = mc.Species(name="C")  
+
+species = {"A":A,"B":B,"C":C}
+
+gas = mc.Phase(name="gas",species=list(species.values()))
+```
+3. Define a mechanism of interest
+
+Through Musica, several different mechanisms can be explored to define reaction rates. Here, we use the Arrhenius equation as a simple example. Reaction parameters as well as additional mechanisms available can be found in our [mechanism configuration documentation](https://ncar.github.io/musica/api/python.html#module-musica.mechanism_configuration). 
+```
+arr1 = mc.Arrhenius(name="A->B", A=4.0e-3, C=50,reactants=[species["A"]], products=[species["B"]], gas_phase=gas)  
+
+arr2 = mc.Arrhenius(name="B->C", A=1.2e-4, B=2.5, C=75, D=50, E=0.5, reactants=[species["B"]], products=[species["C"]], gas_phase=gas)
+
+mechanism = mc.Mechanism(name="test_mechanism", species=list(species.values()),phases=[gas], reactions=[arr1, arr2])
+```
+
+4. Create a box model
+
+To create a box model, including its mechanisms, conditions, length, time, and step times:
+```
+box_model = MusicBox()
+box_model.load_mechanism(mechanism)
+```
+
+In the box model, the initial set of conditions represent the starting environment for the reactions. Both initial and evolving conditions are typically created alongside the creation of the box model:
+```
+box_model.initial_conditions = Conditions(temperature=300.0,pressure=101000.0,species_concentrations={ "A": 1.0,"B": 3.0,"C": 5.0,})
+```
+
+Evolving conditions represent a set of changes to the previous conditions of the box model. To add an evolving condition to the model, where the first float represents the time when the condition evolves:
+```
+box_model.add_evolving_condition(300.0,Conditions(temperature=290.0,pressure=100200.0,species_concentrations={"A": 1.0,"B": 3.0,"C": 10.0,}))
+```
+
+
+```
+box_model.box_model_options.simulation_length = 600.0 # total simulation time
+box_model.box_model_options.chem_step_time = 2 # time step for chemical reaction
+box_model.box_model_options.output_step_time = 6 # time step between each output
+```
+
+5. Solve
+
+To solve and view your newly-created box model, simply run:
+```
+df = box_model.solve()
+print(df)
+```
+
+6. Example Output
+<details><summary>&nbsp;&nbsp;&nbsp;<b>Click to Expand</b></summary>
+
+```python
+|     |   time.s |   ENV.temperature.K |   ENV.pressure.Pa |   ENV.air number density.mol m-3 |   CONC.A.mol m-3 |   CONC.B.mol m-3 |   CONC.C.mol m-3 |
+|----:|---------:|--------------------:|------------------:|---------------------------------:|-----------------:|-----------------:|-----------------:|
```

Comment:
Do we need the `:` here as it shows in the preview of README?

Reply:
Ah thanks for your clarification. I am fine with `:` based on your comments. I just want to make sure it is not a typo.